app/models/workflow_step.rb

```diff
@@ -26,6 +30,10 @@ def as_milestone(xml)
                   version: version)
   end
 
+  def valid_status?
+    errors.add(:status, :invalid_status) unless VALID_STATUS.include? status&.downcase
```

Comment:
why are we downcasing?

Reply:
no particular reason, we just validate as requiring downcased values only